contracts/Automate.sol

```diff
@@ -155,14 +156,72 @@ contract Automate is Gelatofied, Proxied, AutomateStorage, IAutomate {
             success
         );
 
-        emit LogUseGelato1Balance(
-            _oneBalanceParam.sponsor,
+        emit LogUseGelato1Balance(_oneBalanceParam.correlationId);
```

Comment:
I think we should keep the old log the same to avoid more unnecessary refactor on 1balance side?